DiagnosticsExtension/Models/ConnectionStringValidator/ConnectionStringType.cs

```diff
@@ -21,6 +21,9 @@ public enum ConnectionStringType
         RedisCache,
         StorageAccount,
         ServiceBus,
-        EventHubs
-    }
-}
\ No newline at end of file
+        EventHubs,
+        BlobStorageAccount,
```

Comment:
Could you rearrange so that StorageAccount is above or below the other storage ones?